.github/workflows/release.yaml

```diff
@@ -97,11 +104,21 @@ jobs:
             tar -czf ${FILENAME}.tar.gz gator*
             popd
           }
+
           mkdir -p _dist
+
+          i=0
           for os_arch_extension in $PLATFORMS; do
-            build ${os_arch_extension} &
+              build ${os_arch_extension} &
+              pids[${i}]=$!
+              ((i=i+1))
           done
-          wait
+
+          # wait for all pids
+          for pid in ${pids[*]}; do
+              wait $pid
```

Comment:
two thoughts

- wondering if we want a specific timeout here, besides the overall timeout defined on line 19 in this file for all jobs;
- did we want to fail the job if one of the pid's returns non zero?

Reply:
- no specific timeout, just timeout for the pipeline itself
- yea that is the proposed behavior in the PR. we are not doing that today